lib/locales/it.js

```diff
@@ -14,15 +14,15 @@ module.exports = {
     dictionary: {
         'Autoscale': 'Scala automaticamente',                                                               // components/modebar/buttons.js:139
         'Box Select': 'Selezione box',                                                                    // components/modebar/buttons.js:103
-        'Click to enter Colorscale title': 'Clicca per inserire un titolo alla scala di colori',                     // plots/plots.js:437
+        'Click to enter Colorscale title': 'Clicca per inserire un titolo alla scala colori',                     // plots/plots.js:437
```

Comment:
a member of our community said that "scala di colori" would be more correct - can you please let us know?